tests/Config/Guesser/SourceDirGuesserTest.php

```diff
@@ -92,16 +92,15 @@ public function test_it_returns_list_if_contains_array_of_paths_without_src()
         $this->assertSame(['sources', 'libs'], $guesser->guess());
     }
 
-    /**
-     * @expectedException \LogicException
-     * @expectedExceptionMessage autoload section does not match the expected JSON schema
-     */
     public function test_it_throw_invalid_autoload_exception()
     {
         $guesser = new SourceDirGuesser(
             json_decode('{"autoload":{"psr-4": [{"NameSpace\\//": ["sources", "libs"]}]}}')
         );
 
+        $this->expectException(\LogicException::class);
```

Comment:
imo annotations look better :)

Reply:
https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions

PHPUnit documentation does not say that annotations worse or better than code, this is just an alternative. So I am also curious why PHPCSFixer forces to use the code?

Reply:
See [Questioning PHPUnit Best Practices](https://thephp.cc/news/2016/02/questioning-phpunit-best-practices).

Reply:
nice explanation. That moment when I don't understand why it's not in the documentation
thank you